neuraxle/base.py

```diff
@@ -33,8 +33,9 @@
 from collections import OrderedDict
 from copy import copy
 from enum import Enum
-from typing import Tuple, List, Union, Any, Iterable, KeysView, ItemsView, ValuesView, Callable, Optional
+from typing import Tuple, List, Union, Any, Iterable, KeysView, ItemsView, ValuesView, Callable, Optional, Dict
```

Comment:
What are `KeysView, ItemsView, ValuesView` ? I didn't know those existed. Could it be possible to simplify those types?

Reply:
I  didn't add this in this pull requests, and its only for the typings. resolving only for TruncableSteps.keys and TruncableSteps.values.